### PR TITLE
docs: explain what data we need

### PR DESCRIPTION
**Pull request by camjc**
Hopefully we can use this to lobby governments for better sources.

Screen Shot 2020-03-24 at 8 40 26 pm

Screen Shot 2020-03-24 at 8 40 35 pm

### CODE REVIEW

**Comment by chunder** [comment]
Did you mean to include the changes to the linter config and maintainers?

**Comment by camjc** [comment]
Hey, yeah sorry a bit unrelated. Happy to split those out this arvo (I figured they were trivial enough to chuck in here, should have explicitly mentioned, sorry!)

**Comment by chunder** [comment]
Ok, just wanted to check they weren't included by mistake.

**Comment by camjc** [comment]
@chunder Ideally we'd want timestamps too right? Some places are providing data multiple times a day

**Comment by camjc** [comment]
Removed any linting stuff from here; it shouldn't be here and is conflicting anyway.

**Comment by camjc** [comment]
@chunder What are your thoughts on adding time?
- If we don't want to do it we should rename column header to `date`
- If we do want to do it we should revert that commit

**Comment by chunder** [comment]
I feel like this page should be asking for pragmatic, simple things from agencies. If they're sophisticated enough to be comfortable with UTC timestamps, they're probably not looking to us for advice on how to structure their CSVs. They probably are publishing clean data already.

If someone's being directed to read this page, it's because they are making poor decisions like publishing the data as an image of a map, etc. Given that target audience, I feel like we should be offering simple advice on how to produce *better* data. Not perfect data.

So all that said, a simple date column is probably a big win.

File: site/example.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+datetime,   county,        state,  country,  cases,  deaths,  recovered,  tested
```
Comment (camjc) [inline site/example.csv]:
```suggestion
date,       county,        state,  country,  cases,  deaths,  recovered,  tested
```

**Comment by ryanblock** [comment]
Personally, I'd lead with the actual **data** we care more about most, as that's the most important (for obvious reasons).

I'd then advise on presentation layer (which, admittedly, is also quite important!) and other factors.

**Comment by camjc** [comment]
@ryanblock Can you chuck a commit in with what your suggestion is?

**Comment by ryanblock** [comment]
In the interest of not PRing your PR (yo dawg), here's a blob:
```
First, please have a look at our <a href="example.csv" download class="spectrum-Link">example format</a>.<br/>

As the data itself is most important, please publish your total case count, deaths, recovered, and total tests administered.

If you cannot publish in JSON or CSV, at a minimum, please include a HTML <code>&lt;table&gt;</code> with one row per locality at the most granular level you have.<br/>

Have a column for the name of this locality, and a column for each additional data point.<br/>

The best published sources include timestamps, allowing citizens and researchers alike better understand the data you have, over time, within your specific geographic region(s).<br/>
```

**Comment by camjc** [comment]
@ryanblock LGTM, thanks dawg ;)

**Comment by qgolsteyn** [comment]
Let's start with that and adjust if necessary.

**Pull request merged**